### PR TITLE
Add to BASH mentors

### PR DESCRIPTION
**Pull request by machuu**
Add 'machuu' to Exercism BASH mentors.

### CODE REVIEW

**Comment by NobbZ** [comment]
Any reason why you have closed #840 ?

**Comment by machuu** [comment]
I got confused about how to modify the pull request, so I closed and made a new one.
Now I clearly see the 'Commit suggestion' button.

Sorry for any extra work.

File: mentors/bash/m.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "github_username": "machuu",
+    "name": "Matt Hoover",
+    "link_text": "Personal Site",
+    "link_url": "http://hoover.ml",
+    "avatar_url": "http://gravatar.com/avatar/6dcd94787d387411653a88ac408c09dd.png",
+    "bio": "I use BASH daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
```
Comment (Smarticles101) [inline mentors/bash/m.json]: Is it common to have bash written in all caps?
```suggestion
    "bio": "I use Bash daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
```

Comment (Smarticles101) [inline mentors/bash/m.json]: Is it common to have bash written in all caps?
```suggestion
    "bio": "I use Bash daily at work and home, and I think it is a foundational skill for mastering any UNIX-like system."
```

Comment (machuu) [reply mentors/bash/m.json]: I've seen it that way enough times that I started referencing it that way.

After looking at the official README, it should probably be just uppercase B.

Should I update my bio in the Pull Request?

**Comment by Smarticles101** [comment]
Thanks @machuu!

**Pull request merged**